ts/features/bonus/bonusVacanze/screens/AvailableBonusScreen.tsx

```diff
@@ -140,9 +140,8 @@ class AvailableBonusScreen extends React.PureComponent<Props> {
     ]);
 
     if (bpdEnabled) {
-      const bpdHandler = bpdConfig?.program_active
-        ? this.props.startBpdOnboarding
-        : () => this.completedAlert(info.item[getRemoteLocale()].name);
+      const bpdHandler = () =>
+        this.completedAlert(info.item[getRemoteLocale()].name);
```

Comment:
Shouldn't this part also be removed?

Reply:
technically yes, but since it doesn't change much and this section of the codebase is canonically legacy, I figured it would be a better idea to leave that fallback as a safety net until we inevitably drop it all.
What do you think about it? I believe that since it's just three light lines of code, the potential positive outcome outweighs the negatives